### ISSUE TITLE
computeBounds() breaking change in 4.1.0

### ISSUE THREAD

**Issue by AlienGameroom** [treeform/pixie#406]
I’ve been using `computeBounds(font: Font, text: string)` extensively in previous versions to send a text string along with the Font information to determine in advance what size `newImage ()` should be. That proc has changed in 4.1.0. to only accommodate `computeBounds( arrangement: Arrangement,  transform = mat3())`. I have neither an arrangement nor a transform, and I can’t see a solution for just a font and text string. I’m fairly new to Pixie, so apologies in advance if I’m missing something obvious.

**Comment by guzba** [comment]
The previous version of `computeBounds` is now `layoutBounds`.

There exists fonts where the previous version does not work. This is related to some fonts having glyphs for characters that go outside of the glyph typsetting rect (see https://github.com/treeform/pixie/blob/master/tests/fonts/rendered/spans7.png).

Using the previous `computeBounds` works for most fonts, but not all. The new `computeBounds` is actually correct, but more complicated to use to actually get things correct in all cases. Here is how to use it: https://github.com/treeform/pixie/blob/master/tests/test_fonts.nim#L1018

Basically, bounds.wh will work for sizing the image and you'll need to translate the origin a bit to make room for potentially larger glyphs.

**Comment by AlienGameroom** [comment]
Thanks Ryan,
`layoutBounds` works perfectly for what I'm currently doing. I just didn't know where to look :)
I want to thank you and treeform for a fantastic suite of packages. I'm using vmath and pixie, in the process of test driving windy and fidget. I'm also eyeing flatty, netty, zippy and spacy. You guys are Nim Rockstars!
